git-cliff-core/src/changelog.rs

```diff
@@ -39,8 +42,16 @@ pub struct Changelog<'a> {
 impl<'a> Changelog<'a> {
 	/// Constructs a new instance.
 	pub fn new(releases: Vec<Release<'a>>, config: &'a Config) -> Result<Self> {
+		let mut changelog = Changelog::build(releases, config)?;
+		changelog.process_commits();
+		changelog.process_releases();
+		changelog.add_remote_data()?;
+		Ok(changelog)
+	}
+
+	fn build(releases: Vec<Release<'a>>, config: &'a Config) -> Result<Self> {
```

Comment:
Maybe this should be `new` and `new` should be `from_git_releases` or something.

Reply:
I'm okay with both as long as we don't change the public API a lot :)

Reply:
Let's keep it this way then :slightly_smiling_face: